src/OrchardCore.Modules/OrchardCore.Contents/Indexing/IndexingConstants.cs

```diff
@@ -15,5 +15,6 @@ public static class IndexingConstants
         public const string BodyAspectBodyKey = "Content.BodyAspect.Body";
         public const string DisplayTextAnalyzedKey = "Content.ContentItem.DisplayText.Analyzed";
         public const string DisplayTextKey = "Content.ContentItem.DisplayText";
+        public const string FullTextKey = "Content.ContentItem.FullText";
     }
 }
```

Comment:
I forget which files are supposed to have a newline at the end ? Is it .cs files or cshtml ?

Reply:
cs